public/data/textile/processes.json

```diff
@@ -1251,14 +1218,13 @@
       "tre": 0,
       "wtu": 0
     },
-    "name": "Production de coton recyclé (déchets de production)",
     "source": "Custom",
-    "sourceId": null,
+    "sourceId": "Production de coton recyclé (déchets de production)",
     "unit": "kg",
     "waste": 0
   },
   {
-    "categories": ["transformation"],
+    "categories": ["processing"],
     "comment": "Ancien identifiant (12/2024): 9c478d79-ff6b-45e1-9396-c3bd897faa1d.",
     "density": 0,
     "displayName": "Tricotage moyen (mix de métiers circulaire & rectiligne)",
```

Comment:
Please revert previous categories as this makes the build fail

Reply:
Addressed in https://github.com/MTES-MCT/ecobalyse/pull/978/commits/016d60540e9e93586b04bdc267e0cff66f737639